packages/twenty-server/src/engine/core-modules/ai/controllers/ai.controller.spec.ts

```diff
@@ -2,22 +2,29 @@ import { Test, type TestingModule } from '@nestjs/testing';
 
 import { AiService } from 'src/engine/core-modules/ai/services/ai.service';
 import { FeatureFlagService } from 'src/engine/core-modules/feature-flag/services/feature-flag.service';
+import { AIBillingService } from 'src/engine/core-modules/ai/services/ai-billing.service';
 
 import { AiController } from './ai.controller';
 
 describe('AiController', () => {
   let controller: AiController;
   let aiService: jest.Mocked<AiService>;
   let featureFlagService: jest.Mocked<FeatureFlagService>;
+  let aiBillingService: jest.Mocked<AIBillingService>;
 
   beforeEach(async () => {
     const mockAiService = {
       streamText: jest.fn(),
-    };
+      getModel: jest.fn(),
+    } as Partial<jest.Mocked<AiService>>;
```

Comment:
let's avoid the as, why is it needed here? Usually, if the tests do not need to provide more, it's likely that the tested function / class does not need as much as it requests as input. (we can Pick<> in function parameter)